src/index_spec.coffee

```diff
@@ -167,3 +167,27 @@ describe 'alphabetize_keys', ->
           errors = coffeelint.lint notAlphabetical[name]
           expect(errors).to.have.lengthOf 2
           expect(errors[0].rule).to.eql 'alphabetize_keys'
+
+  context 'overrides', ->
+    context 'public methods', ->
+      context 'alphabetical to overrides', ->
+        it 'returns no errors', ->
+          str = notAlphabetical['define class with public instance methods']
+          opts = alphabetize_keys: overrides: ['methodC', 'methodB', 'methodA']
+          errors = coffeelint.lint str, opts
+          expect(errors).to.be.empty
+
+      context 'not alphabetical to overrides', ->
+        it 'returns an error', ->
+          str = notAlphabetical['define class with public instance methods']
+          opts = alphabetize_keys: overrides: ['methodC', 'methodA', 'methodB']
+          errors = coffeelint.lint str, opts
+          expect(errors).to.have.lengthOf 1
+          expect(errors[0].rule).to.eql 'alphabetize_keys'
+
+      context 'not all overrides', ->
```

Comment:
```coffee
context `with other keys`, ->
```
Would be nice to do two examples here. One where the other keys are alphabetical and one where they are not.